### PR TITLE
Allow templates for content representations being rendered as blade t…

### PR DESCRIPTION
**Pull request by fabianmichael**
…emplates.

Before, they where intepreted as plain PHP

### CODE REVIEW

**Comment by fabianmichael** [comment]
🛎

File: src/Template.php [hunk 1]
```diff
@@ -96,6 +96,41 @@ public function file(): ?string
...
+
+        try {
+            // Try the template with type extension in the default template directory.
+            return F::realpath($this->getFilename($name), $this->templatesPath);
+        } catch (Exception $e) {
+            // Look for the template with type extension provided by an extension.
+            // This might be null if the template does not exist.
+            return Kirby::instance()->extension($this->store(), $name);
+        }
+    }
```
Comment (lukasleitsch) [inline src/Template.php]: This section seems to be a duplication of the method `file()`. Is there a way to merge this both methods?

File: src/Template.php [hunk 2]
```diff
@@ -25,13 +25,13 @@ public function __construct(Kirby $kirby, string $name, string $type = 'html', s
...
     {
-        if ($this->isBlade() && $this->hasDefaultType() === true) {
+        if ($this->isBlade()) {
             View::share('kirby', $data['kirby']);
             View::share('site', $data['site']);
             View::share('pages', $data['pages']);
             View::share('page', $data['page']);
 
-            $html = View::make($this->name, $data)->render();
+            $html = View::file($this->bladeFile(), $data)->render();
```
Comment (lukasleitsch) [inline src/Template.php]: Can we use the existing method? I tried it out in a little example and it works for me.

```suggestion
            $html = View::file($this->file(), $data)->render();
```

**Comment by fabianmichael** [comment]
@lukasleitsch I refactored the whole class, looks much leaner now. Please give it a chance. :-)

**Comment by lukasleitsch** [comment]
@fabianmichael Thanks.

**Pull request merged**